src/plugins/chart_expressions/expression_pie/public/plugin.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { CoreSetup, CoreStart, DocLinksStart, ThemeServiceStart } from '../../../../core/public';
+import { ChartsPluginSetup } from '../../../charts/public';
+import { DataPublicPluginStart } from '../../../data/public';
+import { pieLabelsFunction, pieVisFunction } from '../common';
+import { getPieVisRenderer } from './expression_renderers';
+import { ExpressionPiePluginSetup, ExpressionPiePluginStart, SetupDeps, StartDeps } from './types';
+
+/** @internal */
+export interface VisTypePieDependencies {
+  theme: ChartsPluginSetup['theme'];
+  palettes: ChartsPluginSetup['palettes'];
+  getStartDeps: () => Promise<{
+    data: DataPublicPluginStart;
+    docLinks: DocLinksStart;
+    kibanaTheme: ThemeServiceStart;
+  }>;
+}
+
+export interface VisTypePiePluginStartDependencies {
+  data: DataPublicPluginStart;
+}
+
+export class ExpressionPiePlugin {
+  public setup(
+    core: CoreSetup<VisTypePiePluginStartDependencies>,
+    { expressions, visualizations, charts, usageCollection }: SetupDeps
```

Comment:
The visualizations and usageCollection are not used here, let's remove them. We can also remove the usageCollection from the kibana.json required plugins

Reply:
Sure, thanks. Done)